static/src/javascripts-legacy/projects/common/modules/commercial/membership-engagement-banner.js

```diff
@@ -13,12 +13,12 @@ define([
         'common/utils/fastdom-promise',
         'common/modules/experiments/ab',
         'common/modules/experiments/tests/membership-engagement-banner-tests',
-        'common/utils/$',
         'lodash/objects/defaults',
         'lodash/collections/find',
-        'common/views/svgs'
+        'common/views/svgs',
```

Comment:
The `svgs` module imports the whole collection of SVGs, which is massive. If possible, use `svg` and only import those icons you need instead.